packages/icons/src/general/IconCode.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
this got in here because `v2` is not updated with master

Reply:
👍 I'll rebase v2 after I merge the RTL PR.